cmd/ocsp-updater/main.go

```diff
@@ -207,7 +207,10 @@ func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Ti
 	var statuses []core.CertificateStatus
 	_, err := updater.dbMap.Select(
 		&statuses,
-		`SELECT cs.*
+		`SELECT
```

Comment:
Good idea. I wasn't sure if I should put that in this PR but with the splitting it will be irrelevant.